src/site/components/preview/index.njk

```diff
@@ -0,0 +1,29 @@
+---
+pagination:
+  data: examples
+  size: 1
+  alias: example
+  addAllPagesToCollections: false
```

Comment:
`addAllPagesToCollections: false` is actually the default but just made it explicit here.